tracing-subscriber/src/filter/layer_filters/mod.rs

```diff
@@ -470,6 +470,31 @@ impl<L, F, S> Filtered<L, F, S> {
     fn did_enable(&self, f: impl FnOnce()) {
         FILTERING.with(|filtering| filtering.did_enable(self.id(), f))
     }
+
```

Comment:
if we're going to add a `filter_mut` accessor, IMO, we may as well also add a `filter` accessor as well?
```suggestion

    /// Borrows the [`Filter`](crate::filter::Filter) used by this layer.
    pub fn filter(&self) -> &F {
        &self.filter
    }
    
```

Reply:
(it would also be fine to do this in a follow-up PR, but it's a small enough change that we may as well just do it now, I think)